hicommonwealth/commonwealth
# Removed auth guidance and re-require signatures with new addresses

## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/9027

## Description of Changes
- Simplified auth flows, now both sign-in and create-account flows use a similar modal and flows.
- Remove the auth guidance step when a new address is detected
- Removed "create wallet" / "use existing wallet" branches in the create account flow

## "How We Fixed It"
N/A

## Test Plan
- Create SSO/Wallet account with the `Sign-in` flow/button -- verify you get the "welcome user" modal after signup.
- Create SSO/Wallet account with the `Create-Account` flow/button -- verify you get the "welcome user" modal after signup.
- Verify that signing-in with an existing account from both `Sign-in`/`Create-Account` flow, signs you in correctly.
- Ensure no regressions with auth flows.

## Deployment Plan
N/A

## Other Considerations
N/A
